.github/workflow-gen/Program.cs

```diff
@@ -0,0 +1,301 @@
+﻿// Copyright (c) Duende Software. All rights reserved.
+// Licensed under the Apache License, Version 2.0.
+
+using Logicality.GitHub.Actions.Workflow;
+using static GitHubContexts;
+
+var contexts = Instance;
+Component[] components = [
+    new("ignore-this",
+        ["IgnoreThis"],
+        ["IgnoreThis.Tests"],
+        "it"),
+
+    new("bff",
```

Comment:
To be able to run playwright tests, i'll need to be able to install custom tools on the build agent for the BFF.. any thoughts on how we could tackle this in this approach?

Reply:
The purpose of workflow-gen is composable and reusable steps. While reuse of workflows works well in Foss repo because it's all exactly the same there is no hard rule - products are likely to deviate. We'll do what's necessary to minimise friction (which may even mean not using workflow-gen ultimately).